lib/airbrake/better_errors.rb

```diff
@@ -0,0 +1,43 @@
+require 'better_errors'
+
+module BetterErrors
+  # Better Errors' error handling middleware with Airbrake support. Including
+  # this in your middleware stack will report uncatched exceptions to Airbrake
+  # and show a Better Errors error page for exceptions raised below this
+  # middleware.
```

Comment:
This comment sounds like I should include this middleware if I use better_errors. But what it does is hacking better_errors middleware to play nicely with Airbrake.